users/tests/conftest.py

```diff
@@ -1,25 +1,28 @@
 import asyncio
-from typing import Dict
+from typing import Dict, Optional
 
 import pytest
 from asgi_lifespan import LifespanManager
 from httpx import AsyncClient
+from sqlalchemy import select
 from sqlalchemy.ext.asyncio import AsyncConnection, AsyncSession
 
 from app.api.deps import get_session
+from app.core.security import get_password_hash
 from app.core.config import settings
 from app.core.database import engine
 from app.main import app
+from app.models.users import User
 
 
-@pytest.fixture()
+@pytest.fixture(scope="session")
```

Comment:
Ah ok. This can't be session, because the idea is for it to rollback on each test...

Reply:
Yeah.